dossier.py

```diff
@@ -6,6 +6,9 @@
 Date created: 17th April 2021
 """
 
+from metabolism import *
```

Comment:
metabolism should be import in  dossier_example_05_organism_local_efficiency.py 
not dossier itself

Reply:
Okay understood.